components/esp_lvgl_port/esp_lvgl_port.c

```diff
@@ -535,7 +553,7 @@ static void lvgl_port_task(void *arg)
             task_delay_ms = lv_timer_handler();
             lvgl_port_unlock();
         }
-        if (task_delay_ms > lvgl_port_ctx.task_max_sleep_ms) {
+        if ((task_delay_ms > lvgl_port_ctx.task_max_sleep_ms) || (1 == task_delay_ms)) {
```

Comment:
Why?

Reply:
> Why?

```
if(lv_timer_run == false) {
    already_running = false; /*Release mutex*/
    return 1;
}
```
after `lv_timer_enable(false)`,` lv_timer_handler()` will return 1 here.

Reply:
OK, thank you for clarification. It should be changed in future. There should be queue and infinity sleep. Wake only when LVGL or touch need it. I have draft for it and still waiting for LVGL 9, which will have callback for wake, when need it.

Reply:
> OK, thank you for clarification. It should be changed in future. There should be queue and infinity sleep. Wake only when LVGL or touch need it. I have draft for it and still waiting for LVGL 9, which will have callback for wake, when need it.

Okay, currently lvgl 8.3, I don’t see any other low-power input control interface now.
 Looking forward to version 9.0.